File: src/main/java/no/nav/sbl/dialogarena/sak/service/interfaces/SakOgBehandlingService.java
package no.nav.sbl.dialogarena.sak.service.interfaces;

import no.nav.tjeneste.virksomhet.sakogbehandling.v1.informasjon.finnsakogbehandlingskjedeliste.WSSak;

import java.util.List;

public interface SakOgBehandlingService {
    List<WSSak> hentSakerForAktor(String aktorId);
}

File: web/src/main/java/no/nav/modiapersonoversikt/service/sakogbehandling/SakOgBehandlingService.kt
package no.nav.modiapersonoversikt.service.sakogbehandling

import no.nav.modiapersonoversikt.service.sakstema.domain.Behandlingskjede
import no.nav.tjeneste.virksomhet.sakogbehandling.v1.informasjon.finnsakogbehandlingskjedeliste.Sak

interface SakOgBehandlingService {
    fun hentAlleSaker(fnr: String): List<Sak>

    fun hentBehandlingskjederGruppertPaaTema(fnr: String): Map<String, List<Behandlingskjede>>
}
